signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java

```diff
@@ -208,7 +208,10 @@ public boolean onTouchEvent(MotionEvent event) {
             case MotionEvent.ACTION_DOWN:
                 getParent().requestDisallowInterceptTouchEvent(true);
                 mPoints.clear();
-                if (isDoubleClick()) break;
+                if (isDoubleClick()) {
```

Comment:
Shouldn't this be applicable only if the configuration for `clearOnDoubleClick` is `true`?